README.md

```diff
@@ -6,6 +6,9 @@ Below you find some help to commands that you will probaly need.
 
 - [MAN](#man)
 - [PWD](#pwd)
+- [WHOAMI](#whoami)
+- [PASSWD](#passwd)
+- [DATE](#date)
```

Comment:
Since the merging of #5 would overwrite #6 then I added all commands made by @WendellAdriel and commited on #5 to this PR so we would have to merge it only once